### PR TITLE
[WB32 MCU]  Fix some issue.

### PR DESCRIPTION
**Pull request by itarze**
1. Modify exti isr.
2. Fix the issue when the CH_CFG_ST_TIMEDELTA configuration is set to 2.

### CODE REVIEW

**Comment by itarze** [comment]
review the PR, please!

File: demos/WB32/RT-WB32F3G71-RTC/main.c [hunk 1]
```diff
@@ -121,9 +121,9 @@ int main(void) {
   while (true){
     chThdSleepSeconds(2);
     rtcGetTime(&RTCD1, &timespec);
-    chprintf((BaseSequentialStream *)&SERIAL_DEBUG_DRIVER, 
-             "lsi sleep %ds year = %d  month = %d  dstflag=%d  dayofweek = %d  day = %d  millisecond = %d\r\n",
-             RTC_ALARMPERIOD, timespec.year, timespec.month, timespec.dstflag, timespec.dayofweek, timespec.day, timespec.millisecond);
+    // chprintf((BaseSequentialStream *)&SERIAL_DEBUG_DRIVER,
```
Comment (fpoussin) [inline demos/WB32/RT-WB32F3G71-RTC/main.c]: Please remove the code if it's not needed anymore.

**Review by fpoussin — CHANGES_REQUESTED** [review]

File: demos/WB32/RT-WB32F3G71-RTC/main.c [hunk 2]
```diff
@@ -121,9 +121,9 @@ int main(void) {
   while (true){
     chThdSleepSeconds(2);
     rtcGetTime(&RTCD1, &timespec);
-    chprintf((BaseSequentialStream *)&SERIAL_DEBUG_DRIVER, 
-             "lsi sleep %ds year = %d  month = %d  dstflag=%d  dayofweek = %d  day = %d  millisecond = %d\r\n",
-             RTC_ALARMPERIOD, timespec.year, timespec.month, timespec.dstflag, timespec.dayofweek, timespec.day, timespec.millisecond);
+    // chprintf((BaseSequentialStream *)&SERIAL_DEBUG_DRIVER, 
+    //          "lsi sleep %ds year = %d  month = %d  dstflag=%d  dayofweek = %d  day = %d  millisecond = %d\r\n",
+    //          RTC_ALARMPERIOD, timespec.year, timespec.month, timespec.dstflag, timespec.dayofweek, timespec.day, timespec.millisecond);
```
Comment (itarze) [inline demos/WB32/RT-WB32F3G71-RTC/main.c]:
```suggestion
```

**Review by fpoussin — APPROVED** [review]

**Comment by fpoussin** [comment]
Thanks!

**Pull request merged**